liblouis/lou_translateString.c

```diff
@@ -2970,33 +2974,33 @@ resolveEmphasisPassages(EmphasisInfo *buffer, const EmphasisClass *class,
 				pass_word_cnt = 1;
 			}
 		} else /* check if at end of passage */
-			if (in_pass) {
-				if (in_word && !(in_emph_word || last_emph_symbol == i)) {
-				end_passage:
-					in_pass = 0;
+				if (in_pass) {
+			if (in_word && !(in_emph_word || last_emph_symbol == i)) {
+			end_passage:
+				in_pass = 0;
+				if (last_pass_word_end < last_pass_word_start) {
+					last_pass_word_end = i;
+				}
+				/* it is a passage only if the number of words is greater than or
+				 * equal to the minimum length (lencapsphrase / lenemphphrase) */
+				/* if the phrase closing indicator is placed before the last word and
+				 * it was not a whole word, the minimum phrase length is increased */
+				if (!endphraseafter_defined && last_pass_word_end != last_word_end) {
+					pass_word_cnt--;
+				}
+				if (pass_word_cnt >= emphRule[lenPhraseOffset])
+					convertToPassage(pass_start, last_pass_word_end, last_pass_word_start,
+							buffer, class, table, wordBuffer);
+			} else if (i == input->length - 1) {
+				if (pass_word_cnt >= emphRule[lenPhraseOffset]) {
 					if (last_pass_word_end < last_pass_word_start) {
-						last_pass_word_end = i;
-					}
-					/* it is a passage only if the number of words is greater than or
-					 * equal to the minimum length (lencapsphrase / lenemphphrase) */
-					/* if the phrase closing indicator is placed before the last word and
-					 * it was not a whole word, the minimum phrase length is increased */
-					if (!endphraseafter_defined && last_pass_word_end != last_word_end) {
-						pass_word_cnt--;
-					}
-					if (pass_word_cnt >= emphRule[lenPhraseOffset])
-						convertToPassage(pass_start, last_pass_word_end,
-								last_pass_word_start, buffer, class, table, wordBuffer);
-				} else if (i == input->length - 1) {
-					if (pass_word_cnt >= emphRule[lenPhraseOffset]) {
-						if (last_pass_word_end < last_pass_word_start) {
-							last_pass_word_end = input->length;
-						}
-						convertToPassage(pass_start, last_pass_word_end,
-								last_pass_word_start, buffer, class, table, wordBuffer);
+						last_pass_word_end = input->length;
 					}
+					convertToPassage(pass_start, last_pass_word_end, last_pass_word_start,
+							buffer, class, table, wordBuffer);
```

Comment:
It's hard to say whether something changed here and if yes what. Will have to check out the PR

Reply:
Again, I think this might be due to using a different version of clang-format. Can you try to use version 14?

Reply:
If you use `clang-format` 14 then the Check format CI test will probably pass.

Reply:
Fixed by adding curly braces